github/downloader_test.go

```diff
@@ -0,0 +1,205 @@
+package github
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/src-d/metadata-retrieval/github/store"
+	"github.com/stretchr/testify/require"
+
+	"golang.org/x/oauth2"
+)
+
+func TestOnlineRepositoryDownload(t *testing.T) {
```

Comment:
please add a couple of lines of doc here, [some rules](https://blog.golang.org/godoc-documenting-go-code)